content/10.methods.md

```diff
@@ -113,7 +113,7 @@ wru implements methods described in Imai and Khanna [@doi:10.1093/pan/mpw001] to
 The underlying data used for prediction are derived from the US Census.
 We used only the surname of author or honoree to make predictions via the *predict_race()* function.
 However, in the case of names that were not observed in the census, the function's behavior was to use the average demographic distribution from the census.
-We modified the function to return a status denoting that results were inconclusive instead.
+We modified the function to return a status denoting that results were inconclusive (NA) instead.
```

Comment:
```suggestion
To avoid this suboptimal imputation, we modified the function to return a status denoting that results were inconclusive (NA) instead.
```